README.md

```diff
@@ -155,10 +155,11 @@ Retired Staff
 - Cathy J. Fitzpatrick [cathyjf] - Development, Sysadmin
 - Hugh Gordon [V4] - Research (game mechanics), Development
 - Juanma Serrano [Joim] - Development, Sysadmin
+- Leonardo Julca [Slayer95] - Development
 
 Major Contributors
 
-- Bär Halberkamp [bumbadadabum] - Development
+- [HoeenHero] - Development
```

Comment:
@HoeenCoder I say this to everyone: You should be credited by real name.

There are a few reasons I prefer real name:

1. It makes it easier for you to actually point to this in résumés and job interviews and say "that's me"
2. It makes the project look more professional
3. The entire point of crediting is to tell people who you are, not telling people who you are kind of defeats the purpose

Reply:
Alright, list me as Andrew Werner bumba